src/Randomizer.App/Controls/MetroidControlsPanel.xaml

```diff
@@ -0,0 +1,115 @@
+                  Grid.Column="1"
+                  Grid.Row="0"
+                  Content="Auto Run" />
+        </Grid>
+        <controls:LabeledControl Text="Item cancel button behavior:"
+                                 ToolTip="Adjusts how the item cancel button works:&#xD;&#xA;Vanilla - Deselects the current item&#xD;&#xA;Hold - Hold button to keep super missiles/power bombs selected&#xD;&#xA;Toggle - Press to select super missiles/power bombs if nothing is selected">
+          <ComboBox SelectedItem="{Binding ItemCancelBehavior}"
+            ItemsSource="{Binding Source={local:EnumBindingSource {x:Type options:ItemCancelBehavior}}}"
+            HorizontalAlignment="Left"
+            MinWidth="75" />
+        </controls:LabeledControl>
+      </StackPanel>
+    </GroupBox>
+    <GroupBox Header="Button Mappings"
+                        Padding="0,5,0,0"
+                        Margin="0,0,0,10">
+      <StackPanel>
+        <TextBlock Margin="0,0,0,10" TextWrapping="Wrap">Button mappings are based on the default SNES controls and do not account for different emulator button mappings.</TextBlock>
+        
+        <controls:LabeledControl Text="Shoot:">
+          <ComboBox Name="ComboBox_Shoot"
+                    SelectionChanged="ComboBox_ButtonMappingChanged"
+                    SelectedItem="{Binding Shoot}"
+                    ItemsSource="{Binding Source={local:EnumBindingSource {x:Type options:MetroidButton}}}"
+                    HorizontalAlignment="Left"
+                    MinWidth="75" />
+        </controls:LabeledControl>
+
+        <controls:LabeledControl Text="Jump:">
+          <ComboBox Name="ComboBox_Jump"
+                    SelectionChanged="ComboBox_ButtonMappingChanged"
+                    SelectedItem="{Binding Jump}"
+                    ItemsSource="{Binding Source={local:EnumBindingSource {x:Type options:MetroidButton}}}"
+                    HorizontalAlignment="Left"
+                    MinWidth="75" />
+        </controls:LabeledControl>
+
+        <controls:LabeledControl Text="Dash:">
+          <ComboBox Name="ComboBox_Dash"
+                    SelectionChanged="ComboBox_ButtonMappingChanged"
+                    SelectedItem="{Binding Dash}"
+                    ItemsSource="{Binding Source={local:EnumBindingSource {x:Type options:MetroidButton}}}"
+                    HorizontalAlignment="Left"
+                    MinWidth="75" />
+        </controls:LabeledControl>
+
+        <controls:LabeledControl Text="Item Select:">
+          <ComboBox Name="ComboBox_ItemSelect"
+                    SelectionChanged="ComboBox_ButtonMappingChanged"
+                    SelectedItem="{Binding ItemSelect}"
+                    ItemsSource="{Binding Source={local:EnumBindingSource {x:Type options:MetroidButton}}}"
+                    HorizontalAlignment="Left"
+                    MinWidth="75" />
+        </controls:LabeledControl>
+
+        <controls:LabeledControl Text="Item Cancel:">
```

Comment:
It might be convenient to change this from "Item Cancel" to something more appropriate based on the selection above, e.g. "Item Cancel / Hold to select ammo" or "Item Cancel / Toggle ammo". Maybe the selection should even be next to this control since they're related?

Reply:
I moved it and auto run to be next to the adjacent buttons, so that everything makes more sense contextually.